docs/sources/flow/when-to-use-flow/opentelemetry-collector.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Learn how to Flow compares to the OpenTelemetry Collector
+menuTitle: OpenTelemetry Collector
+title: OpenTelemetry Collector
+weight: 100
+---
+
+# Comparing Flow with the OpenTelemetry Collector
+
+## Which one of the two should you use? 
+
+Flow may be a better choice if:
+* You prefer River syntax over YAML.
+* You would like to collect "profile" telemetry - Flow can do this via Pyroscope components.
+* You would like to process Prometheus or Loki telemetry without converting it to OTLP.
+
+The Collector may be a better choice if:
+* You are already using the Collector, and do not need any of the features of Flow.
+* You prefer YAML syntax over River.
+
+Below you can find more details on similarities and differences between the Collector and Flow.
+
+## Similarities
+
+### Many Collector components are available in Flow
+
+Flow contains many of the components available inside the Collector:
+* For example, the OTLP exporter is available as `otelcol.exporter.otlp`.
+* The Grafana Agent development team is actively working on adding more of the Collector's components to Flow.
+* If a Flow user needs a Collector feature to be made available in the Agent, the Agent development team could work on implementing it.
```

Comment:
These aren't components available inside the Collector.

I'd suggest we convert the bullets into sentences and just make a paragraph out of this information.

Reply:
I agree! I reworded it and it looks better to me now.